Sources/Plasma/CoreLib/CMakeLists.txt

```diff
@@ -88,6 +88,7 @@ target_link_libraries(
         $<$<AND:$<CONFIG:Debug>,$<BOOL:${USE_VLD}>>:VLD::VLD>
     PRIVATE
         "$<$<PLATFORM_ID:Darwin>:-framework CoreFoundation>"
+        "$<$<PLATFORM_ID:Darwin>:-framework Accelerate>"
```

Comment:
Since `Accelerate/Accelerate.h` is now being used in a header (therefore being consumed by non-CoreLib libraries), this needs to be a `PUBLIC` link library.